components/butterfly/protocols/swim.proto

```diff
@@ -51,6 +51,7 @@ message Service {
   optional string pkg = 9;
   optional bytes cfg = 10;
   optional SysInfo sys = 12;
+  optional string channel = 13;
```

Comment:
I missed @reset 's last message I'll look at refactoring and having something else feed the event server.